logger regression existing in attestation service

During the update to variadic options in NewAttestationService() a regression was added that made the logger not to be available making the go-routine to fail.

```
"level":"info","ts":1678228411.2617261,"kind":"server","component":"grpc","operation":"/controlplane.v1.AttestationService/Store","args":"***","code":0,"reason":"","stack":"","latency":0.006055486}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1756b90]

goroutine 32545 [running]:
github.com/go-kratos/kratos/v2/log.(*Helper).Infow(...)
        /home/runner/go/pkg/mod/github.com/go-kratos/kratos/v2@v2.5.3/log/helper.go:81
github.com/chainloop-dev/chainloop/app/controlplane/internal/service.uploadSBOMsToDependencyTrack(0xc00109d7c0)
```

- Make sure the logger is initialized
- Add a default logger for the parent struct to prevent future cases

--- a/app/controlplane/internal/service/attestation.go
+++ b/app/controlplane/internal/service/attestation.go
@@ -72,7 +72,7 @@ type NewAttestationServiceOpts struct {
 
 func NewAttestationService(opts *NewAttestationServiceOpts) *AttestationService {
 	return &AttestationService{
-		service:                 newService(),
+		service:                 newService(opts.Opts...),
 		wrUseCase:               opts.WorkflowRunUC,
 		workflowUseCase:         opts.WorkflowUC,
 		attestationUseCase:      opts.AttestationUC,

--- a/app/controlplane/internal/service/service.go
+++ b/app/controlplane/internal/service/service.go
@@ -17,6 +17,7 @@ package service
 
 import (
 	"context"
+	"io"
 
 	"github.com/chainloop-dev/chainloop/app/controlplane/internal/usercontext"
 	"github.com/chainloop-dev/chainloop/internal/servicelogger"
@@ -53,7 +54,9 @@ func loadCurrentUserAndOrg(ctx context.Context) (*usercontext.User, *usercontext
 }
 
 func newService(opts ...NewOpt) *service {
-	s := &service{}
+	s := &service{
+		log: log.NewHelper(log.NewStdLogger(io.Discard)),
+	}
 
 	for _, opt := range opts {
 		opt(s)

